packages/blockchain-wallet-v4-frontend/src/modals/Xlm/XlmCreateAccountLearn/index.js

```diff
@@ -45,7 +45,7 @@ const BackIcon = styled(Icon)`
 class XlmCreateAccountLearn extends React.PureComponent {
   onClose = () => {
     this.props.closeAll()
-    this.props.modalActions.showModal('@MODAL.SEND.XLM', {
+    this.props.modalActions.showModal('SEND_XLM_MODAL', {
```

Comment:
maybe to move such thing to be from const or from some enum and then just reuse it all around, also this could be done for all modalEnhancer like:

```

export default modalEnhancer('SHOW_XPUB_MODAL')(ShowXPubContainer)

replace with 

enum MODAL_ENHANCER {
SHOW_XPUB_MODAL = 'SHOW_XPUB_MODAL',
...
}

export default modalEnhancer(MODAL_ENHANCER.SHOW_XPUB_MODAL)(ShowXPubContainer)

```

Reply:
Can we do this later? This PR is too long already